commands/box/ctl/root.go

```diff
@@ -91,51 +82,55 @@ func init() {
 			Short: "Get the block header for a hash",
 			Run:   getBlockHeaderCmdFunc,
 		},
+
 		&cobra.Command{
-			Use:   "getinfo",
-			Short: "Get info about the local node",
-			Run: func(cmd *cobra.Command, args []string) {
-				fmt.Println("getinfo called")
-			},
-		},
-		&cobra.Command{
-			Use:   "getnetworkinfo [network]",
-			Short: "Get the basic info and performance metrics of a network",
-			Run: func(cmd *cobra.Command, args []string) {
-				fmt.Println("getnetworkinfo called")
-			},
-		},
-		&cobra.Command{
-			Use:   "searchrawtxs [address]",
-			Short: "Search transactions for a given address",
-			Run: func(cmd *cobra.Command, args []string) {
-				fmt.Println("searchrawtx called")
-			},
-		},
-		&cobra.Command{
-			Use:   "signmessage [message] [optional publickey]",
-			Short: "Sign a message with a publickey",
-			Run:   signMessageCmdFunc,
+			Use:   "viewblockdetail [blockhash]",
+			Short: "Get the raw blockInformation for a block hash",
+			Run:   getBlockDetailCmdFunc,
 		},
 		&cobra.Command{
-			Use:   "validateaddress [address]",
-			Short: "Check if an address is valid",
-			Run:   validateMessageCmdFunc,
+			Use:   "getinfo",
```

Comment:
??

Reply:
remove validateaddrss to wallet and  add netinfo command to Ctl